lib/LINZ/BdeDatabase.pm

```diff
@@ -541,6 +541,7 @@ sub streamDataToTempTable
 
     # See https://metacpan.org/pod/DBD::Pg#COPY-support
     my @buf;
+    $datafh->binmode(":encoding(UTF-8)");
```

Comment:
Would setting this when the FH is first opened be a better place? e.g in BDEUpload-> _OpenDataFile https://github.com/linz/linz-bde-uploader/blob/master/lib/LINZ/BdeUpload.pm#L1193

Reply:
I'm still thinking if if isn't better to only decode when passing the value to pg_copydata instead.
And... I've to check why the test is now failing..